src/main/java/io/vertx/ext/stomp/PermittedOptions.java

```diff
@@ -0,0 +1,98 @@
+/*
+ *  Copyright (c) 2011-2015 The original author or authors
+ *  ------------------------------------------------------
+ *  All rights reserved. This program and the accompanying materials
+ *  are made available under the terms of the Eclipse Public License v1.0
+ *  and Apache License v2.0 which accompanies this distribution.
+ *
+ *       The Eclipse Public License is available at
+ *       http://www.eclipse.org/legal/epl-v10.html
+ *
+ *       The Apache License v2.0 is available at
+ *       http://www.opensource.org/licenses/apache2.0.php
+ *
+ *  You may elect to redistribute this code under either of these licenses.
+ */
+
+package io.vertx.ext.stomp;
+
+import io.vertx.codegen.annotations.DataObject;
+import io.vertx.core.json.JsonObject;
+
+
+/**
+ * Specify a match to allow for inbound and outbound traffic.
+ *
+ * @author <a href="http://escoffier.me">Clement Escoffier</a>
+ */
```

Comment:
Using https://github.com/vert-x3/vertx-web/blob/master/vertx-web/src/main/java/io/vertx/ext/web/handler/sockjs/BridgeOptions.java would require a dependency on vertx web. At some point we should move this "common" configuration to core.